components/cards/UserAvailability.tsx

```diff
@@ -61,4 +62,7 @@ const styles = StyleSheet.create({
     borderRadius: 2,
     marginTop: 12,
   },
+  activity: {
+    lineHeight: 20,
+  },
```

Comment:
we don't need this style since the text for the activity has fontsize `15`. So here it can use `CustomText` 's default size is `base`, which has `fontsize: 15 and lineheight: 24`
![image](https://github.com/user-attachments/assets/551bb5a9-c816-456c-af35-9bf76aac995f)